help/de/Help_ch_Account-Actions.xml

```diff
@@ -718,69 +668,85 @@
+                    <quote>Online-Preisabfrage installieren</quote>.
+                  </para>
+                </listitem>
+              </varlistentry>
 
-              <listitem>
-                <para>Trifft keine der obigen Anleitungen auf Ihre Umgebung zu, so können Sie die folgende alternative,
-                  allgemeinere Anweisungen probiern:
-                </para>
+              <varlistentry>
+                <term>&mac;</term>
 
-                <orderedlist spacing="compact">
-                  <listitem>
-                    <para>Schließen Sie alle &app; Anwendungen, die Sie gerade ausführen.
-                    </para>
-                  </listitem>
+                <listitem>
+                  <para>Wenn Sie &app; mit Hilfe des Installationsprogramms auf der &app; Homepage installiert haben, ist
+                    <application>Finance::Quote</application> bereits installiert.
+                  </para>
+                </listitem>
+              </varlistentry>
 
-                  <listitem>
-                    <para>Suchen Sie den Ordner, in dem &app; installiert ist, indem Sie nach <quote>gnc-fq-update</quote>
-                      (ohne die Anführungszeichen) suchen.
-                    </para>
-                  </listitem>
-
-                  <listitem>
-                    <para>Wechseln Sie in dieses Verzeichnis, öffnen Sie eine Root-Shell und führen Sie den Befehl
-                      <code>gnc-fq-update</code> aus. Dadurch wird eine Perl CPAN-Update-Sitzung
-                      gestartet, die ins Internet geht und das Modul
-                      <application>Finance::Quote</application> und seine Abhängigkeiten auf Ihrem
-                      System installiert. Das Programm gnc-fq-update ist interaktiv, bei den meisten
-                      Systemen sollten Sie jedoch die erste Frage mit <quote>Nein</quote>
-                      beantworten können: <quote>Sind Sie bereit für die manuelle Konfiguration?
-                      [ja]</quote> und die Aktualisierung wird automatisch fortgesetzt von diesem
-                      Punkt an.
-                    </para>
-                  </listitem>
-                </orderedlist>
-              </listitem>
-            </itemizedlist>
+              <varlistentry>
+                <term></term>
+
+                <listitem>
+                  <para>Trifft keine der obigen Anleitungen auf Ihre Umgebung zu, so können Sie die folgende alternative,
+                    allgemeinere Anweisungen probiern:
+                  </para>
+
+                  <orderedlist spacing="compact">
+                    <listitem>
+                      <para>Schließen Sie alle &app;-Anwendungen, die Sie gerade ausführen.
+                      </para>
+                    </listitem>
+
+                    <listitem>
+                      <para>Lokalisieren Sie den Ordner, in dem &app; installiert ist, indem Sie nach <code>gnc-fq-update</code>
```

Comment:
code ist die inline-Variante von programlisting für Programmiercode. Anscheinend wie bei syntaxhighlight im wiki [mögliche Sprachen](https://pygments.org/docs/lexers/)

Du könntest also language="sh" (posix) bzw. powershell, posh, ps1, psm1 probieren.
Daneben hat docbook auch andere Attribute. Hier würde ich vielleicht einfach userinput nehmen.

Reply:
ok. ich habe mich für `<code language="shell">` entschieden.

Reply:
Wird das auch schön bunt in yelp?

Reply:
nö, "bunt" wird da nichts, die Schriftart und -größe ändern sich nur im Vergleich zum normalen Text. In Yelp erkenne ich keine Änderungen, in HTML wird bei manchen Befehlen der Text fett. Und die beiden Befehle `command` und `filename` bringen optischen auch keine Verbesserungen.